### ISSUE TITLE
Can it use unix domain sockets? 

### ISSUE THREAD

**Issue by nailgilaziev** [reactiverse/reactive-pg-client#100]
App server and pg server often placed in one unix machine. Communication via unix domain sockets significantly improve performance. Does it use it internally?

**Comment by vietj** [comment]
yes it is possible to use it, perhaps you can try and contribute a doc section ?

the `PgClient` is really a Vert.x `NetClient` and the actual `PgConnectOptions` extends `NetClientOptions`.

https://vertx.io/docs/vertx-core/java/#_domain_sockets

**Comment by BillyYccc** [comment]
I'm glad to investigate and make a contribution for this if no one is going to handle it.

**Comment by vietj** [comment]
feel free to investigate @BillyYccc
